js/client_exporter/image_creator.js

```diff
@@ -481,9 +481,7 @@ function createFilter(element) {
 
         switch(attr.result.value) {
             case "gaussianBlurResult":
-                if(attr.stdDeviation) { // T511738, IE10
```

Comment:
@roman-simionov 
```if(attr.stdDeviation) {```
Is this condition is actual for IE10 only?

Reply:
I have not found any browser compatibility table for this. We need check it in all supported browsers

Reply:
IE 10.0.9200.16384 - IE10 bug
IE 11.0.9600.18098 - it works
IE 11.48.17134.0 - it works
Edge 42.17143.1.0 - it works
Chrome 61, 66 - it works
FF 60.0.1 - it works
Safari 11.1 - it works
Opera 53.0 - it works
Safari (ios 11.4) - no changes
Chrome 66 (Android 8) - it works